lib/ui/screens/profile/profile_view.dart

```diff
@@ -1,12 +1,125 @@
+                padding: _screenPadding,
+                children: <Widget>[
+                  _buildAvatar(),
+                  const SizedBox(height: _paddingBetweenAvatarName),
+                  _buildNameAndDescription(),
+                  const SizedBox(height: _paddingBetweenDescriptionField),
+                  const Column(
+                    crossAxisAlignment: CrossAxisAlignment.start,
+                    children: <Widget>[
+                      CustomTextFormField(
+                        labelText: 'Email',
+                        hintText: 'Edit your email',
+                        icon: Icons.email_outlined,
+                      ),
+                      SizedBox(height: _spaceBetweenGroups),
+                      CustomTextFormField(
+                        labelText: 'Phone Number',
+                        hintText: 'Edit your phone number',
+                        icon: Icons.smartphone_outlined,
+                      ),
+                      SizedBox(height: _spaceBetweenGroups),
+                      PasswordField(
+                        labelText: 'Password',
+                        hintText: 'Edit your password',
+                      ),
+                      SizedBox(height: _emptySpaceAfterPassword),
+                    ],
+                  ),
+                ],
+              ),
+            ),
+            CustomButton(text: 'Logout', onPressed: () {}),
+          ],
+        ),
+      );
+
+  Widget _buildAvatar() => Builder(
+        builder: (BuildContext context) => FittedBox(
+          fit: BoxFit.scaleDown,
+          child: Stack(
+            alignment: Alignment.center,
+            children: <Widget>[
+              CircleAvatar(
+                radius: _avatarSize,
+                child: Image.asset(
+                  'assets/images/dummy_profile_pic.png',
+                ),
+              ),
+              Positioned(
+                right: _positionAvatarIcon,
+                bottom: 0,
+                child: Container(
+                  width: _containerSize,
+                  height: _containerSize,
+                  decoration: BoxDecoration(
+                    color: Theme.of(context).colorScheme.outline,
+                    shape: BoxShape.circle,
+                  ),
+                  child: IconButton(
+                    icon: Icon(
+                      Icons.create_outlined,
+                      color: Theme.of(context).colorScheme.onPrimary,
+                    ),
+                    onPressed: () {
+                      // TODO: implementation
+                    },
+                    padding: const EdgeInsets.all(_paddingAvatarIcon),
+                  ),
+                ),
+              ),
+            ],
+          ),
+        ),
+      );
+
+  Widget _buildNameAndDescription() => Builder(
+        builder: (BuildContext context) => Column(
+          children: <Widget>[
+            Text(
+              dummyUser.name,
```

Comment:
**suggestion:** Use actual data from the model

Reply:
done